coreneuron/mech/mod2c_core_thread.h

```diff
@@ -1,8 +1,8 @@
 #ifndef mod2c_core_thread_h
 #define mod2c_core_thread_h
 
-#include "coreneuron/nrnoc/multicore.h"
-#include "coreneuron/nrnoc/nrnoc_ml.h"
+#include "coreneuron/sim/multicore.h"
+#include "mechanism.h"
```

Comment:
If NEURON introduces same one, would be better to have this included as `coreneuron/mechanism.h`?

Reply:
we had said to put it into `coreneuron/mech` maybe not a bad place?